TidepoolKit Example/Support/Extensions/Sample.swift

```diff
@@ -64,6 +110,21 @@ struct Sample {
                                         energy: TFoodDatum.Nutrition.Energy(350, .kilocalories))
         }
 
+        fileprivate static var cgmSettingsAlerts: TCGMSettingsDatum.Alerts {
+            let snooze = TCGMSettingsDatum.Alerts.Snooze(30, .minutes)
+            return TCGMSettingsDatum.Alerts(enabled: true,
+                                            urgentLow: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 55, units: .milligramsPerDeciliter, snooze: snooze),
+                                            urgentLowPredicted: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 55, units: .milligramsPerDeciliter, snooze: snooze),
+                                            low: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 70, units: .milligramsPerDeciliter, snooze: snooze),
+                                            lowPredicted: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 65, units: .milligramsPerDeciliter, snooze: snooze),
+                                            high: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 180, units: .milligramsPerDeciliter, snooze: snooze),
+                                            highPredicted: TCGMSettingsDatum.Alerts.LevelAlert(enabled: true, level: 200, units: .milligramsPerDeciliter, snooze: snooze),
+                                            fall: TCGMSettingsDatum.Alerts.RateAlert(enabled: true, rate: 3, units: .milligramsPerDeciliterPerMinute, snooze: snooze),
```

Comment:
This seems like a strange value for a `fall`. Is `+3` really the expected slope of the curve?

Reply:
The `3` represents the setting the user chooses in the Dexcom UI which is either `2` or `3` mg/dL/min. It is the absolute value of actual slope.

Reply:
I suspect Dexcom G6 app converts this value from what is displayed in the UI based on my knowledge of the SDK. Tidepool Loop needs to provide a negative value when constructing the actual alert. Is basing this on the UI correct? Wouldn't it be better to provide the actual value instead of its absolute value? Or is the absolute value always provided for such things to the backend? I see potential for confusion.

Reply:
Is there a reason that the absolute value is needed here?

Reply:
Well, this is already a published, public API and there is already data in the backend database using this format. So, I don't think we can (easily) change this going now without requiring a backend change and a database migration. Plus, potentially updating any client apps that already use this data.

In short, while not idea, I think this has to stay as-is. I don't think it is worth the effort to change at this point.